README.md

```diff
@@ -15,8 +15,8 @@ This is the best blog app ever app, it was build with love using RoR.
 1. Clone the project using `git clone`.
 2. Open the project folder with VSCode or any Editor.
 3. Open terminal and navigate to the project folder.
-4. Type `irb`.
-5. Copy the code from files and run it.
+4. Run `bundle install`.
+5. Run `rails db:create`.
```

Comment:
- [ ] It would be better if we are able to run the app locally given the instructions given in the README, However I am unable to run the app, unless we migrate the database and run or start the server `rails db:migrate` and `rails s`

Reply:
done, thanks!